appconfiguration/client/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -72,10 +72,10 @@
     /**
      * Creates a builder that can configure options for the ConfigurationAsyncClient before creating an instance of it.
      *
-     * @return A new {@link ConfigurationAsyncClientBuilder} to create a ConfigurationAsyncClient from.
+     * @return A new {@link ConfigurationClientBuilder} to create a ConfigurationAsyncClient from.
      */
-    public static ConfigurationAsyncClientBuilder builder() {
-        return new ConfigurationAsyncClientBuilder();
+    public static ConfigurationClientBuilder builder() {
+        return new ConfigurationClientBuilder();
     }
```

Comment:
Please re-review the GitHub issue - these builder methods should no longer exist.